lib/collection.js

```diff
@@ -1634,7 +1645,7 @@ Collection.prototype.findOneAndUpdate = function(filter, update, options, callba
  * @return {Promise} returns Promise if no callback passed
  * @deprecated use findOneAndUpdate, findOneAndReplace or findOneAndDelete instead
  */
-Collection.prototype.findAndModify = function(query, sort, doc, options, callback) {
+Collection.prototype.findAndModify = deprecate(function(query, sort, doc, options, callback) {
```

Comment:
If a deprecated function has deprecated options, should we use both util.deprecate and deprecateOptions or is util.deprecate enough?

Reply:
`util.deprecate` is good enough IMO